requirements.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   repository: file://./charts/cp-kafka-rest
   condition: cp-kafka-rest.enabled
 - name: cp-kafka-connect
-  version: 0.1.0
+  version: 0.2.0
```

Comment:
no need to update the version here. I update the version when updating umbrella chart

Reply:
Got you.
I reverted it.